comfy_cli/command/run.py

```
@@ -44,16 +94,26 @@ def execute(workflow: str, host, port, wait=True, verbose=False, local_paths=Fal
         raise typer.Exit(code=1)
 
+    with open(workflow_name, encoding="utf-8") as f:
+        raw_workflow = json.load(f)
+
+    if is_ui_workflow(raw_workflow):
+        pprint("[yellow]Detected UI-format workflow, converting via server's /workflow/convert...[/yellow]")
+        try:
+            workflow = convert_ui_workflow_via_server(raw_workflow, host, port, timeout)
+        except WorkflowConverterUnavailable:
+            _print_converter_unavailable_help()
+            raise typer.Exit(code=1)
```

Pylint (4.0.5)

[warning] Consider explicitly re-raising using 'except WorkflowConverterUnavailable as exc' and 'raise typer.Exit(code=1) from exc'

(W0707)